x-pack/plugin/esql/qa/testFixtures/src/main/resources/where-match.csv-spec

```diff
@@ -0,0 +1,68 @@
+###############################################
```

Comment:
I think I'd just call this file `match`

Reply:
we will also have a match command which is why this is not just called `match`.